ledger/light/handle/set_result.go

```diff
@@ -0,0 +1,102 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package handle
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/insolar/insolar/insolar/flow"
+	"github.com/insolar/insolar/insolar/payload"
+	"github.com/insolar/insolar/insolar/record"
+	"github.com/insolar/insolar/ledger/light/proc"
+	"github.com/pkg/errors"
+)
+
+type SetResult struct {
+	dep     *proc.Dependencies
+	message payload.Meta
+	passed  bool
+}
+
+func NewSetResult(dep *proc.Dependencies, msg payload.Meta, passed bool) *SetResult {
+	return &SetResult{
+		dep:     dep,
+		message: msg,
+		passed:  passed,
+	}
+}
+
+func (s *SetResult) Present(ctx context.Context, f flow.Flow) error {
+	msg := payload.SetResult{}
+	err := msg.Unmarshal(s.message.Payload)
+	if err != nil {
+		return errors.Wrap(err, "failed to unmarshal SetResult message")
+	}
+
+	calc := proc.NewCalculateID(msg.Result, flow.Pulse(ctx))
+	s.dep.CalculateID(calc)
+	if err := f.Procedure(ctx, calc, true); err != nil {
+		return err
+	}
+	reqID := calc.Result.ID
```

Comment:
`resID`

Reply:
I'll do it in  next PR about ActivateObject record